bip-taproot.mediawiki

```diff
@@ -236,6 +236,22 @@ def taproot_output_script(internal_pubkey, script_tree):
 
 [[File:bip-taproot/tree.png|frame|This diagram shows the hashing structure to obtain the tweak from an internal key ''P'' and a Merkle tree consisting of 5 script leaves. ''A'', ''B'', ''C'' and ''E'' are ''TapLeaf'' hashes similar to ''D'' and ''AB'' is a ''TapBranch'' hash. Note that when ''CDE'' is computed ''E'' is hashed first because ''E'' is less than ''CD''.]]
 
+To spend this output using script ''D'', the control block would contain the following data in this order:
+
+<source>
+[Leaf version of D + control bits][internal key P][C][E][AB]
```

Comment:
Lower case p is more consistent with the algorithm description, same below, because it implicitly means the serialized point.

Reply:
nit: Perhaps first element "control byte with leaf version" would be a bit more clear?

Reply:
A bit unclear what the `[...]` notation actually means. Could also just use a `,` separator. Or list

1.
2.
3.

Reply:
Thanks, yes it should be lower case p, will update.

Not sure about the enumerated list, maybe angle brackets like in the [bip141 examples](https://github.com/bitcoin/bips/blob/master/bip-0141.mediawiki#examples)?

I like "control byte with leaf version", will update.

Reply:
Ok, updated lmk what you think.

Reply:
Looks good.